Library/Fantasy/Fantasy Advantages.adq

```diff
@@ -19,6 +19,35 @@
 				"points": 2
 			}
 		},
+		{
+			"id": "379c5b53-3fa5-4728-8a3d-be657650cea6",
+			"type": "trait",
+			"name": "Named Possession",
+			"reference": "F132",
+			"tags": [
+				"Perk",
+				"Social",
+				"Supernatural"
+			],
+			"base_points": 5,
+			"calc": {
+				"points": 5
+			}
+		},
+		{
+			"id": "d3c15c2e-9bcd-4b9c-83c5-e75444a97a50",
+			"type": "trait",
+			"name": "Weapon Bond",
+			"reference": "F132",
+			"tags": [
+				"Perk",
```

Comment:
Here, either.

Reply:
Sorry, my bad, I'm new to GURPS and for some reason kept thinking perks and quirks were 5 points advantages and disadvantages.

They are indeed perks, but I should correct their cost in 1 point.
Will do it right now and push the changes!